sections/appendix/variational_inference.tex

```diff
@@ -11,6 +11,20 @@ \subsection{\acrlong{vi}}
 In \autoref{fig:vi}, we can see a visualization of this where we want to optimize the variational parameters to minimize the distance to the true posterior distribution.
 In \autoref{fig:vi}, $q(z; \nu)$ is a family of variational distributions over the latent variables which are different instances of the hidden variables $z$.
 Our goal is to fit the variational parameters $\nu$ to estimate the hidden variables that are closest to $p(z|\textbf{x})$
+In every iteration, we try to minimize the $q$ function in \autoref{eq:q_func} 
+\begin{equation}\label{eq:q_func}
+	\hat{q}(z) = arg \min_{\theta} K\!L(q_\theta, p(z|\textbf{x}))
+\end{equation}
+We still cannot compute $p(z|\textbf{x})$, nor the Kullback-Leibler Divergence, but we can rearrange the $K\!L$ function to make it possible to estimate the $K\!L$.
+\begin{align}
+	K\!L(q_\theta || p(z|\textbf{x}) &= \mathop{\mathbb{E}}_{q}[\log\frac{q(\textbf{z})}{p(\textbf{z}|\textbf{x})}]\\
+	\downdownarrows \\
+	K\!L(q_\theta || p(z|\textbf{x}) &= \log p(\textbf{x}) - (-\mathop{\mathbb{E}}_{q}[\log\frac{q(\textbf{z})}{p(\textbf{z},\textbf{x})}]) \\
+	\downdownarrows \\
+	K\!L(q_\theta || p(z|\textbf{x}) &= \log p(\textbf{x}) - \mathcal{L}(q)
+\end{align}
+where $\mathcal{L}(q)$ is known as the ELBO.
+Since $p(\textbf{x})$ is a constant, when we maximize the ELBO, the $K\!L(q_\theta, p(z|\textbf{x})$ which brings us closer to the true posterior.
```

Comment:
```suggestion
Since $p(\textbf{x})$ is a constant, when we maximize the ELBO, the $K\!L(q_\theta, p(z|\textbf{x}))$ decreases which brings us closer to the true posterior.
```